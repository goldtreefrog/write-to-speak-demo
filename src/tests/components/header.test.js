import React from "react";
import ReactDOM from "react-dom";
import { shallow } from "enzyme";
import Header from "./../../components/header.js";

it("renders without crashing and matches snapshot", () => {
  const wrapper = shallow(<Header writing={{ isEditing: false, editingPage: "edit", snippetsAvail: true }} />);
  expect(wrapper).toMatchSnapshot();
});
