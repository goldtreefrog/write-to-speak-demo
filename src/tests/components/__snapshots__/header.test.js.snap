// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing and matches snapshot 1`] = `
<header
  className="app-header"
  role="banner"
>
  <Link
    replace={false}
    to="/"
  >
    <img
      alt="logo"
      className="app-logo"
      src="wts-pencil2.svg"
    />
    <h1
      className="app-title"
    >
      Write to Speak
    </h1>
  </Link>
  <nav>
    <Link
      replace={false}
      to="/"
    >
      Home
    </Link>
    <Link
      replace={false}
      to="/write"
    >
      Write
    </Link>
    <Link
      replace={false}
      to="/talk"
    >
      Talk
    </Link>
  </nav>
</header>
`;
